src/lang/ar.js

```diff
@@ -1,4 +1,3 @@
-/* eslint quotes: 0 */
```

Comment:
I have a feeling Serge is going to re-add this comment back when the translations come back...